spec/unpacker_spec.rb

```diff
@@ -662,6 +664,69 @@ def flatten(struct, results = [])
       end
     end
 
+    context 'freeze' do
```

Comment:
Could you add a test for `MessagePack.unpack()` ? We can pass options for Unpackers like `MessagePack.unpack(binary, symbolize_keys: true)`

Reply:
Sure.

Reply:
Done.